src/main/java/com/capgemini/molveno/controller/api/OrderController.java

```diff
@@ -0,0 +1,47 @@
+package com.capgemini.molveno.controller.api;
+
+import com.capgemini.molveno.model.Order;
+import com.capgemini.molveno.repository.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/order")
+public class OrderController {
+
+    @Autowired
+    private OrderRepository orderRepository;
```

Comment:
Gebruik de OrderService, niet de repository. De repository is alleen bedoeld voor de service.